### ISSUE TITLE
Training uvit

### ISSUE THREAD

**Issue by Jihyeon-Je** [thu-ml/unidiffuser#38]
Hi, thanks for the work! I'm trying to train the u-vit part from the original u-vit codebase, but I'm having difficulty modifying it to fit the rest of the unidiffuser framework. What loss/model config did you use to train the u-vit? Is it possible to get the training code for the u-vit portion?

**Comment by miquel-espinosa** [comment]
Will the training codes and training instructions for unidiffuser be released? @baofff @NieShenRuc @chenxwh 
Is huggingface team planning on creating some training scripts / guidelines for training unidiffuser models? @sayakpaul @hysts 
Thanks,

**Comment by sayakpaul** [comment]
We're not planning to. However, we used UViT for aMUSEd and you can find all details about how it was trained in the following report: https://huggingface.co/papers/2401.01808
